src/tables.jl

```diff
@@ -236,8 +236,8 @@ Encode an integer into a `BitArray`.
 """
 function int2bitarray(k::Integer; pad::Int = 8)
     res = BitArray{1}(undef, pad)
-    @inbounds for i in pad:-1:1
-        res[i] = k & 1
+    for i in pad:-1:1
```

Comment:
The `@inbounds` macro works on the entire block, thus these are actually more-or-less equivalent.